src/testing/flow_solver_cases/periodic_cube_flow.h

```diff
@@ -30,6 +30,9 @@ class PeriodicCubeFlow : public FlowSolverCaseBase<dim,nstate>
     /// Destructor
     ~PeriodicCubeFlow() {};
 
+    /// Integrates over the entire domain
+    double integrate_over_domain(DGBase<dim, double> &dg,const std::string integrate_what) const;
```

Comment:
integrate_what could be an enum instead of a string.

Reply:
Addressed in most recent commit